flopy/utils/gridintersect.py

```diff
@@ -212,7 +169,7 @@ def intersect(
         return_all_intersections=False,
         contains_centroid=False,
         min_area_fraction=None,
-        shapely2=True,
```

Comment:
Maybe worth keeping the `shapely2` parameter for now to avoid breakage, and warn if provided?

Reply:
Done.